tests/ert_tests/ert3/data/test_transformers.py

```diff
@@ -98,8 +98,9 @@ async def test_atomic_transformation_input(
         await transmitter.transmit_record(record_in)
```

Comment:
When I think about the file I guess we should rename it to `test_transformations.py`